src/main/java/seedu/duke/CommandParser.java

```diff
@@ -269,6 +251,44 @@ public Command parseMiscellaneous(String userInputLowerCase) throws HotelLiteMan
         return userCommand;
     }
 
+    /**
+     * Parses the user-provided view-related command and creates the relevant Command object.
+     *
+     * @param userInputLowerCase User input to be parsed and turned into a delete-related Command object.
+     * @return The relevant Command object created based on the user input.
+     * @throws HotelLiteManagerException If there is an error in user input that prevents it from being parsed into
+     *                                   the relevant Command object.
+     */
+    public Command parseView(String userInputLowerCase) throws HotelLiteManagerException {
```

Comment:
After changing the .replace to .replace first,  do remember to implement the check conditions to catch the duplicate commands in a separate issue.